.github/workflows/api-db-audit.yml

```
@@ -0,0 +1,89 @@
+  api-db-audit:
+    runs-on: ubuntu-latest
+    env:
+      SQL_SA_PASSWORD: ${{ secrets.SQL_SA_PASSWORD }}
+    services:
+      sqlserver:
+        image: mcr.microsoft.com/mssql/server:2022-latest
+        env:
+          SA_PASSWORD: ${{ env.SQL_SA_PASSWORD }}
+          ACCEPT_EULA: "Y"
+        ports: ["1433:1433"]
+        options: >-
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)